test/verify/check-system-services

```diff
@@ -1309,6 +1309,67 @@ class TestTimers(MachineCase):
         m.execute("timedatectl set-time '2036-04-10 04:10:00'")
         wait_systemctl_timer("2036-04-10 04:26")
         self.assertIn("2036-04-10 04:26", m.execute("systemctl list-timers"))
+        # creates a new minutely timer that runs at *:*:05 and at *:*:26
+        b.wait_visible("#create-timer")
+        b.click('#create-timer')
```

Comment:
`b.click()` already does `b.wait_visible()` so no need to do it like this. Feel free to drop the `wait_visible()` call

Reply:
Removed from Minutely and custom.  this persists on the rest of the timer tests.